project/src/main/java/com/google/sps/servlets/EventRankerServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlets;
```

Comment:
Please also add test for this file. You might not need to test every detail of rankEvents, just test if a normal case of calling rankEvents, and a case that user is not present.

Reply:
Done, moved to new file